client/src/parent/TextField.tsx

```diff
@@ -0,0 +1,23 @@
+import { fieldToTextField, TextFieldProps } from "formik-material-ui";
+import { useCallback } from "react";
+import MuiTextField from "@material-ui/core/TextField";
+
+export function TextField(props: TextFieldProps) {
```

Comment:
@marZeu8 Ich habe eine separate TextField Komponente erstellt. Diese habe ich anhand deiner Implementation in den beiden Dialogen übernommen. Vielleicht kann man diese Komponente in den Dialogen ja auch wiederverwenden?

Reply:
Nice, ich denke schon ja :)